pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1649,7 +1596,7 @@ func (og *operationGenerator) GenerateExpandInUseVolumeFunc(
 				resizeOptions.DevicePath = volumeToMount.DevicePath
 				dmp, err := volumeAttacher.GetDeviceMountPath(volumeToMount.VolumeSpec)
```

Comment:
This is definitely the wrong path for a raw block volume. It doesn't even exist.

Reply:
should be fixed now.